daml/Marketplace/Trading/Confirmation.daml

```diff
@@ -78,4 +78,4 @@ template Confirmation
             )
 
           create Settlement.SettlementInstruction with ..
-
+          
```

Comment:
You can avoid this easily in VS Code by setting a flag which removes trailing spaces on save.

https://stackoverflow.com/questions/30884131/remove-trailing-spaces-automatically-or-with-a-shortcut

Reply:
good to know, thanks